tests/runner.js

```diff
@@ -3,5 +3,24 @@
 
 import 'babel-polyfill';
 
+if (process.env.COVERAGE === 'enabled') {
+  // generate the coverage reports
+  after((done) => {
+    let istanbul = require('babel-istanbul');
+    let collector = new istanbul.Collector();
+    let reporter = new istanbul.Reporter();
+    let sync = false;
+
+    collector.add(global.__coverage__);
+
+    reporter.add('text');
+    reporter.addAll([ 'lcov', 'clover' ]);
+    reporter.write(collector, sync, function() {
+      console.log('All reports generated');
+      done();
```

Comment:
If you set up coveralls you can get it to fail when coverage drops below a certain overall percentage threshold or by a defined percentage. It will comment on PRS too.